mcux/scripts/pinctrl/gen_soc_headers.py

```diff
@@ -117,8 +120,8 @@ def main():
 
     data_version = get_pack_version(temp_dir.name)
     print(f"Found data pack version {data_version}")
-    if round(data_version) != 14:
-        print("Warning: This tool is only verified for data pack version 14, "
+    if round(data_version) != 16:
```

Comment:
Thanks for updating this- can you make this change within a separate commit though?